glTF-Toolkit/inc/GLTFLODUtils.h

```diff
@@ -30,7 +30,7 @@ namespace Microsoft::glTF::Toolkit
         /// </summary>
         /// <returns>The primary GLTF Document with the inserted LOD node.</returns>
         /// <param name="docs">A vector of glTF documents to merge as LODs. The first element of the vector is assumed to be the primary LOD.</param>
-        static GLTFDocument MergeDocumentsAsLODs(const std::vector<GLTFDocument>& docs);
+        static GLTFDocument MergeDocumentsAsLODs(const std::vector<GLTFDocument>& docs, const std::vector<std::wstring>& relativePaths);

```

Comment:
This should have an option without this parameter that assumes everything is in the same directory